data/mods/Aftershock/items/tools.json

```diff
@@ -275,6 +275,19 @@
     "qualities": [ [ "CUT", 2 ], [ "SAW_M", 10 ], [ "BUTCHER", -50 ] ],
     "flags": [ "ALWAYS_TWOHAND", "MESSY", "TRADER_AVOID", "NONCONDUCTIVE" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "afs_oxygen_candle",
+    "symbol": "*",
+    "color": "light_gray",
+    "name": { "str": "oxygen candle" },
+    "description": "An ignitable device that produces oxygen through an exothermic chemical reaction.  Often stocked as the last  backup in case of total failure in life support machinery. You could activate it, but that would just waste it.",
```

Comment:
```suggestion
    "description": "An ignitable device that produces oxygen through an exothermic chemical reaction.  Often stocked as the last backup in case of total life support failure.  You could activate it, but that would just waste it.",
```